### PR TITLE
No concurrency

### PR DESCRIPTION
**Pull request by bouda1**
concurrency directory removed. It is no more needed.

### CODE REVIEW

File: inc/com/centreon/clib/version.hh [hunk 1]
```diff
@@ -20,10 +20,10 @@
 # define CC_CLIB_VERSION_HH
 
 // Compile-time values.
-# define CENTREON_CLIB_VERSION_MAJOR  19
-# define CENTREON_CLIB_VERSION_MINOR  10
+# define CENTREON_CLIB_VERSION_MAJOR  20
+# define CENTREON_CLIB_VERSION_MINOR  04
 # define CENTREON_CLIB_VERSION_PATCH  0
-# define CENTREON_CLIB_VERSION_STRING "19.10.0"
+# define CENTREON_CLIB_VERSION_STRING "20.04.0"
```
Comment (SylvestreG) [inline inc/com/centreon/clib/version.hh]: I think this file is generated and should not be commited.

File: inc/com/centreon/handle_manager_posix.hh [hunk 2]
```diff
@@ -0,0 +1,74 @@
+/*
```
Comment (SylvestreG) [inline inc/com/centreon/handle_manager_posix.hh]: why not renaming this file handle_manager.hh, and remove the inderection ?

Comment (bouda1) [reply inc/com/centreon/handle_manager_posix.hh]: Yes, it was planned :smile:

Comment (bouda1) [reply inc/com/centreon/handle_manager_posix.hh]: Done.

**Pull request merged**